web/tsconfig.base.json

```
@@ -20,7 +20,9 @@
       "@humansignal/datamanager": ["libs/datamanager/src/index.js"],
       "@humansignal/editor": ["libs/editor/src/index.js"],
       "@humansignal/frontend-test/*": ["libs/frontend-test/src/*"],
-      "@humansignal/ui": ["libs/ui/src/index.ts"]
+      "@humansignal/ui": ["libs/ui/src/index.ts"],
+      "@humansignal/ui/*": ["libs/ui/src/*"],
```

are both these declarations required?
```suggestion
      "@humansignal/ui/*": ["libs/ui/src/*"],
```

---

it didn't work for me with just one declaration, so I kept both